### PR TITLE
Patch: Style onboardjs modal

### PR DESCRIPTION
**Pull request by ECWireless**
- [x] Add environment variables
- [x] Leave all farms and "view more" open by default
- [x] Style `onboardjs` modal
- [x] Add insufficient funds modal

### CODE REVIEW

File: src/components/Modals/ConfirmTransaction/components/InsufficientFunds/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import 'styled-components/macro';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as CloseSvg } from 'assets/icons/close.svg';
+import { ReactComponent as PiggyBank } from 'assets/icons/piggy_bank.svg';
+
+import { StyledCloseContainer, StyledModalContainer, StyledPiggyBankContainer } from '../../components';
```
Comment (Evalir) [inline src/components/Modals/ConfirmTransaction/components/InsufficientFunds/index.tsx]: I see we're using absolute imports elsewhere—should we do so here as well?

File: src/components/Modals/ConfirmTransaction/index.tsx [hunk 2]
```diff
@@ -64,151 +66,157 @@ const ConfirmTransaction: React.FC = () => {
...
+            )}
+            <StyledDepositButtonContainer>
+              {actionType === 'deposit' ? (
+                <button onClick={onDeposit}>
+                  <DepositButtonActiveSvg />
+                </button>
+              ) : (
+                <button onClick={onWithdraw}>
+                  <WithdrawButtonActiveSvg />
+                </button>
```
Comment (Evalir) [inline src/components/Modals/ConfirmTransaction/index.tsx:213]: I just noticed... I think we have a bunch of buttons that have SVGs and nothing else in them—this isn't very accessible. We should have at least some visually-hidden text to do this. The [Visually Hidden](https://reach.tech/visually-hidden) component from Reach UI can accomplish this. We should do this for all buttons that have this exact situation. We can probably do this in another PR though 😄

File: src/index.css [hunk 3]
```diff
@@ -152,3 +152,43 @@ code {
   font-family: 'Montserrat', sans-serif;
   font-style: normal;
 }
+
+.bn-onboard-modal {
+  background: rgba(0, 0, 0, 0.6) !important;
+  z-index: 10000 !important;
```
Comment (Evalir) [inline src/index.css:158]: Do we need the important? and do we need the huge z-index value?

**Review by Evalir — COMMENTED** [review]
Overall looking pretty good, just noticed some a11y/css-related things:

File: src/components/Modals/ConfirmTransaction/components/InsufficientFunds/index.tsx (hunk 1 above)
Comment (ECWireless) [reply src/components/Modals/ConfirmTransaction/components/InsufficientFunds/index.tsx]: I had it relative for fairly nested components, but yeah may as well keep it consistent: Fixed in d1b2ddbe77ae21483d3b5bee3f2ab397753e27bd

File: src/components/Modals/ConfirmTransaction/index.tsx (hunk 2 above)
Comment (ECWireless) [reply src/components/Modals/ConfirmTransaction/index.tsx:213]: Issue moved to #18

File: src/index.css (hunk 3 above)
Comment (ECWireless) [reply src/index.css:158]: I currently have each layer's z-index separated by factors of 10, so I've just done the same thing here to put the backdrop above the navigation. Think I should reduce z-index values all around? 

And it seems that `onboardjs`'s backdrop is for some reason the only thing that can't be modified without adding `!important` to the `css`. Haven't yet found any other way around it.

Comment (Evalir) [reply src/index.css:158]: Yeah, I had forgotten we were using an external lib... It's aight then, that's usually what follows with these kinds of things haha. And nope, the z-index is fine! Just wanted to make sure it wasn't some "nuclear" value.

**Review by Evalir — APPROVED** [review]
Looks good!
